gdcm_wrapper.cc

```diff
@@ -4,35 +4,37 @@
 
 using namespace std;
 
-struct PixelData c_decode_single_frame_compressed(
-    char * i_buffer_ptr,
+struct PixelData c_decode_frames(
+    char **i_buffer_ptr,
+    size_t *i_buffer_lens,
     size_t i_buffer_len,
-    unsigned int width,
-    unsigned int height,
+    unsigned int dims[3],
```

Comment:
This was already an issue before the pull request, but it is recommended to use [fixed width integer types](https://en.cppreference.com/w/cpp/types/integer) when interfacing with Rust. Maybe one of us can adjust these signatures in a separate PR.

Reply:
[d747cd3](https://github.com/pevers/gdcm-rs/pull/7/commits/d747cd3d1f3f2ad0222178b6c986de7cf4d3af93)